test/lib/test_services.cc

```diff
@@ -197,7 +197,12 @@ future<> test_env::do_with_async(noncopyable_function<void (test_env&)> func, te
     });
 }
 
-data_dictionary::storage_options make_test_object_storage_options() {
+std::optional<data_dictionary::storage_options> make_test_object_storage_options() {
+    if (!::getenv("SCYLLA_TEST_ENV")) {
+        testlog.info("Skipping object-storage test case. Check test/pylib/minio_server.py for an example of how to configure the environment for it to run.");
+        return {};
```

Comment:
The message suggests that you need to need to run minio as shown in that example, but in fact you also need to set SCYLLA_TEST_ENV or this won't work ;-)

At first reading, I wanted to ask you not to use a new variable at all, and just check if the actual minio variables are set up. On second reading, I realized there's also a benefit to your approach - it means that test.py claims it has everything set up, and now the test cases can complain if something is missing - instead of sliently skipping the test, which can cause us to suddenly start skipping various tests in Jenkins without even realizing it because something broken in test.py.